pkg/apis/apps/install/install.go

```diff
@@ -37,7 +37,7 @@ func Install(groupFactoryRegistry announced.APIGroupFactoryRegistry, registry *r
 	if err := announced.NewGroupMetaFactory(
 		&announced.GroupMetaFactoryArgs{
 			GroupName:                  apps.GroupName,
-			VersionPreferenceOrder:     []string{v1beta1.SchemeGroupVersion.Version, v1beta2.SchemeGroupVersion.Version},
+			VersionPreferenceOrder:     []string{v1beta2.SchemeGroupVersion.Version, v1beta1.SchemeGroupVersion.Version},
```

Comment:
Does this change the default storage version? Will it work against <1.8 kubectl?
cc @kubernetes/sig-api-machinery-misc